### DISCUSSION TITLE
Psalmi Feria in Paschal Octave

### DISCUSSION THREAD

**Discussion by Augustinus-Altovadensis** [DivinumOfficium/divinum-officium#3200] [Q&A] [answered]
Laudétur Jesus Christus!
I have come up with an interesting problem recently. In Cistercian office, during the Paschal Octave, only Monday and Tuesday have the Sunday psalms, the rest has Ferial psalms, but the rest remains the same like the other versions - the Office basically copies the Paschal Sunday, with the exception of Benedictus/Magnificat antiphons and Collect.

Now my question: is there any way to override the "Psalms Dominica" rule from the Tempora/Pasch0-[1-6].txt file without changing the code? 
(I'm copying the Monastic Office version, I use a separate folder TemporaCist, that copies - and heavily edits - the TemporaM folder)
I mean, I don't have problem changing the code, although it involves a lot of searching and trial and error, I just don't want to reinvent the wheel...

Thanks a lot, God bless!

**Comment by mbab** [comment, marked as answer]
it looks hard :-(
if You remove `Psalmi Dominica` from `[Rule]` it still exist in **commune rule**  due to `ex Pasc0-0` in `[Rank]`
if You also change in `[Rank]` from `ex` to `vide` You will got proper psalms but with Ferial antiphons
if You need Pascal ants You need to copy text from of `[Ant Laudes]` with proper psalms
`Angelus autem ... alleluia;;50` etc.
other section probably also should be copied or linked

Happy to see Cistercian version alive.

\+

**Reply by Augustinus-Altovadensis** [reply]
Thank you, I thought that. So the simplest solution will be just to copy & paste those four days, and break the "ex" rule altogether. 

Yes, Cistercian version is alive, I'm working mainly on Lauds and Vespers, as those are the hours that I actually use every day, and most of the _Horæ minores_ work as well. All I need to do is to sort out the commemorations (in progress), mainly break the "Commemoratio ad Laudes tantum", which is rare in the Cistercian office, and is used only on High Feasts (_Festa Sermonis_). And I suppose, I will have to sort out the Antiphons for the Minor Hours. And maybe Matins, but most of that should be fine.

And then, only a "small" detail - fill in the Calendar and all the proper Cistercian texts. And that will take a while.

**Comment by FAJ-Munich** [comment]
The developments for Monastic 1930 pushed yesterday, entail a mechanism for Octaves (e.g. Ascension) to combine Octave Antiphones with detail psalter for matins. 
This should be the same way it needs to be applied for Cistercian Pschal Octave, shouldn't it?

**Reply by Augustinus-Altovadensis** [reply]
Well, probably. I haven't had the time yet to review it, and given all the changes in upstream that go against what I have already done, there is some chance that I even won't... However, I have quite simply implemented a rule `Psalmi feria`, which helped me reach the same goal. So Octaves are not a problem anymore.
